src/preferences.cpp

```diff
@@ -496,3 +512,13 @@ void Preferences_API::setMiniHist(const bool &enabled)
 {
 	preferencePanel->mini_hist_enabled = enabled;
 }
+
+bool Preferences_API::getDigitalDecoders() const
+{
+	return preferencePanel->digital_decoders_enabled;
+}
+
+void Preferences_API::setDigitalDecoders(const bool &enabled)
```

Comment:
There is no benefit in passing a constant reference to a bool variable. It just makes the function signature a little bit longer.